src/integrationTest/java/uk/gov/hmcts/reform/professionalapi/CreateMinimalOrganisationTest.java

```diff
@@ -317,8 +320,28 @@ public void returns_200_when_valid_email_is_passed() {
                     professionalReferenceDataClient.createOrganisation(organisationCreationRequest.build());
             assertThat(response.get("http_status")).isEqualTo("201 CREATED");
         });
+    }
 
+    @Test
+    public void return_400_invalid_organisation_with_invalid_email() {
+        String prefix = UUID.randomUUID().toString();
+        Set<String> paymentAccounts = new HashSet<>();
+        paymentAccounts.add("PBA1234567");
+        OrganisationCreationRequest organisationCreationRequest = anOrganisationCreationRequest()
+                .name("some-org-name")
+                .paymentAccount(paymentAccounts)
+                .superUser(aUserCreationRequest()
+                        .firstName("some-fname")
+                        .lastName("some-lname")
+                        .email(String.format("s@-somewhere.com", prefix))
+                        .jurisdictions(createJurisdictions())
+                        .build())
+                .contactInformation(Arrays.asList(aContactInformationCreationRequest().addressLine1("addressLine1").build()))
+                .build();
 
+        Map<String, Object> response = professionalReferenceDataClient.createOrganisation(organisationCreationRequest);
 
+        String orgIdentifierResponse = (String) response.get("organisationIdentifier");
```

Comment:
If you're not doing anything with `orgIdentifierResponse` is there a need for it?

Reply:
Ok . i.e existing test case copied from some other class to here . so i will remove it.